workflows/VGP-assembly-v2/Scaffolding-HiC-VGP8/Scaffolding-HiC-VGP8-tests.yml

```diff
@@ -56,5 +54,5 @@
     Pretext Map After HiC scaffolding:
       asserts:
         has_size: 
-          value : 893000
-          delta: 50000
+          value : 1200000
```

Comment:
That does seem like a huge difference, did you look at the result and confirm this makes sense ?

Reply:
Yes, the outputs of filter and merge had a lower coverage than the ones generated by bwa-mem2 -SP , so the alignment and pretext files are a bit larger now.